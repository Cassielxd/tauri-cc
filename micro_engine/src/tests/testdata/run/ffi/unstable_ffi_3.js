Deno[Deno.internal].core.opAsync("op_ffi_call_ptr_nonblocking", null, {
  name: null,
  parameters: [],
  result: "void",
}, []);
