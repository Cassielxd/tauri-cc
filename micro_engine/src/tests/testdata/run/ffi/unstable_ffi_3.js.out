Unstable API 'Deno.UnsafeFnPointer#call'. The --unstable flag must be provided.
